src/locale/es.json

```diff
@@ -22,6 +22,7 @@
     "housing-action-title": "Movilización para la defensa de la vivienda",
     "housing-action-description": "Donde <strong class='color-rent-strike'>acciones para la justicia</strong> están ocurriendo o han ocurrido.",
     "housing-action-form": "Complete para presentar la información sobre: Organización, resistencia y ayuda mutua en forma de huelgas de alquiler, recuperación de viviendas, ocupaciones y más. También llene si busca las redes de solidaridad y recursos para ayudar a cancelar el alquiler.",
+    "submit": "Envíe datos",
     "resources": {
       "title": "Recursos de apoyo",
       "description": "Aquí hay algunos recursos para obtener ayuda si usted está en necesidad.",
```

Comment:
line 28 with key "description" is no longer needed, it must be deleted

Reply:
Fixed!